addons/website/tests/test_website_visitor.py

```diff
@@ -185,7 +185,7 @@ def _prepare_linked_visitor_data(self):
 
 class WebsiteVisitorTests(WebsiteVisitorTestsCommon):
 
-    readonly_enabled = False
+    registry_readonly_enabled = False
```

Comment:
There are some places where you use the `set_registry_readonly_mode` but not here, is there a reason?

Reply:
Not really, I just forgot about it, but one is a global flag while the other is meant to be used during tests, the whole thing is a bit iffy because of previous use of registry_test_mode

Reply:
I will make the flag private and use the method everywhere as to make sure it is not changed without having the registry patched.